src-cpp/app/CMakeLists.txt

```diff
@@ -5,31 +5,31 @@ add_library(app
     global/singleton.cpp
     modules/shared/section_item.cpp
     modules/shared/section_model.cpp
-    modules/startup/controller.cpp
-    modules/startup/module.cpp
-    modules/startup/view.cpp
-    modules/startup/onboarding/controller.cpp
-    modules/startup/onboarding/item.cpp
-    modules/startup/onboarding/model.cpp
-    modules/startup/onboarding/module.cpp
-    modules/startup/onboarding/view.cpp
-    modules/startup/login/controller.cpp
-    modules/startup/login/item.cpp
-    modules/startup/login/model.cpp
-    modules/startup/login/module.cpp
-    modules/startup/login/view.cpp
+    modules/startup/controller_startup.cpp
```

Comment:
CMake can handle that (no compilation conflicts for src files with the same name) without prefixes, would it be possible for qmake as well?